compiler/data/kphp-json-tags.h

```diff
@@ -0,0 +1,100 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2020 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#pragma once
+
+#include <string>
+
+#include "common/wrappers/string_view.h"
+
+class ClassData;
+class PhpDocComment;
```

Comment:
It's very strange to see ClassData here and in function prototypes.
I suppose, you want to include "compiler/data/data_ptr.h", drop these 2 forward declarations, and use ClassPtr instead of const CllassData& below

Reply:
No, it is exactly what I wanted to do. Working with classes, but not with pointer - like wrappers

Reply:
Can you describe this decision?
Nowhere in our code `ClassData` and `FunctionData` are used directly, we always use `ClassPtr` and `FunctionPtr`

Reply:
Rewritten kphp-json-tags completely to support inheriting false values